scripts/create_interactive_predicate_learning_plots.py

```diff
@@ -28,10 +28,15 @@
     ("PERC_SOLVED", "perc_solved"),
     ("NUM_TRANSITIONS", "num_transitions"),
     ("QUERY_COST", "query_cost"),
+    ("NUM_EXECUTION_FAILURES", "num_execution_failures"),
+    ("NUM_PLANNING_FAILURES", "num_planning_failures"),
 ]
 
 DERIVED_KEYS = [("perc_solved",
-                 lambda r: 100 * r["num_solved"] / r["num_test_tasks"])]
+                 lambda r: 100 * r["num_solved"] / r["num_test_tasks"]),
+                ("num_planning_failures",
```

Comment:
is it guaranteed that all non-solved tasks will be one of these two types of failures? are we ignoring things like code crashes?

Reply:
why not just have another entry in the results for num_planning_failures?